module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,17 @@
+#include "sort.hpp"
+
+#include <algorithm>
+
+bool compare(const std::string& first_string, const std::string& second_string) {
+    return first_string.size() < second_string.size();
+}
+
+std::deque<std::string> lengthSort(std::forward_list<std::string> input_list) {
+    std::deque<std::string> result_deque;
+    input_list.sort();
+    for (std::string element : input_list) {
+        result_deque.emplace_back(element);
+    }
```

Comment:
You can do that -> `std::deque<std::string> result_deque {list.begin(), list.end()};`

Reply:
Done in latest commit, thanks :)